### PR TITLE
Document PHPUnit 6 support

### PR DESCRIPTION
**Pull request by antograssiot**

### CODE REVIEW

File: en/appendices/3-4-migration-guide.rst [hunk 1]
```diff
@@ -4,6 +4,11 @@
 CakePHP 3.4 is an API compatible upgrade from 3.3. This page outlines the
 changes and improvements made in 3.4.
 
+Minimum PHP 5.6 Required
```
Comment (ravage84) [inline en/appendices/3-4-migration-guide.rst]: https://github.com/cakephp/cakephp/blob/3.4.0/composer.json#L21 👍

File: en/contributing/code.rst [hunk 2]
```diff
@@ -13,7 +13,7 @@ setup. You'll need the following software:
 
 * Git
 * PHP |minphpversion| or greater
-* PHPUnit 3.7.0 or greater
```
Comment (ravage84) [inline en/contributing/code.rst]: 😼

File: en/development/testing.rst [hunk 3]
```diff
@@ -23,6 +23,9 @@ To install PHPUnit with Composer:
 
 .. code-block:: bash
 
+    $ php composer.phar require --dev phpunit/phpunit:"^5.7|^6.0"
```
Comment (ravage84) [inline en/development/testing.rst]: This is not reflected here:
https://github.com/cakephp/cakephp/blob/3.4.0/composer.json#L34

Comment (markstory) [inline en/development/testing.rst]: No because the change is on master..

**Pull request merged**